server/src/routes/project.ts

```diff
@@ -0,0 +1,67 @@
+import { PrismaClient } from "@prisma/client";
+import express from "express";
+
+export const projectRoutes = express.Router();
+const prisma = new PrismaClient();
+
+projectRoutes.route("/").get(async (req, res) => {
+  try {
+    // todo: find better way of retrieving numeric fields
+    ["id", "expo", "round", "table"].forEach(key => {
+      if (req.query.hasOwnProperty(key))
+        // @ts-ignore
+        req.query[key] = parseInt(req.query[key]);
+    });
+    const projects = await prisma.project.findMany({ where: req.query });
+    res.status(200).send(projects);
```

Comment:
instead of `.send`, use `.json` so that the response type is parsed as `application/json`. Also, please update the rest of the usages in the code too